release-please-config.json

```diff
@@ -0,0 +1,195 @@
+{
+  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+  "release-type": "simple",
+  "include-component-in-tag": true,
+  "include-v-in-tag": false,
+  "changelog-path": "docs/history.md",
+  "packages": {
+    "src/Google.Cloud.Functions.Framework": {
+      "component": "Google.Cloud.Functions.Framework",
+      "changelog-path": "/docs/history.md",
+      "extra-files": [
+        {
+          "type": "xml",
+          "path": "/src/CommonProperties.xml",
+          "xpath": "//Project/PropertyGroup/Version"
+        }
+      ]
+    },
+    "src/Google.Cloud.Functions.Hosting": {
+      "component": "Google.Cloud.Functions.Hosting",
+      "extra-files": [
```

Comment:
Is there any way of globbing this? I can easily see us adding a new example but forgetting to update the version numbers. (I've already got a shell script to do the update, if release-please can run that as part of the process...) It's at https://github.com/GoogleCloudPlatform/functions-framework-dotnet/blob/main/update-project-references.sh

Reply:
There's currently not a way to glob extra files, but I opened https://github.com/googleapis/release-please/issues/1619 to investigate